### PR TITLE
feat: adding callback timestamps for sms_messages_transactional

### PR DESCRIPTION
**Pull request by richard-yak**
Adding on callback timestamps for sms_messages_transactional.

Deployment Checklist
- [x] Remove Staging PPT test 
- [x] Test on Staging
- [x] Add back Staging PPT test
- [ ] Run migration script on prod

### CODE REVIEW

**Comment by tan-zx** [comment]
are there any integration tests to add? or no need

**Comment by richard-yak** [comment]
hmmm can, lemme update the integration tests

**Review by tan-zx — APPROVED** [review]

File: backend/src/sms/models/sms-message-transactional.ts [hunk 1]
```diff
@@ -55,4 +55,16 @@ export class SmsMessageTransactional extends Model<SmsMessageTransactional> {
 
   @Column({ type: DataType.STRING, allowNull: true })
   errorCode: string | null
+
+  @Column({ type: DataType.DATE, allowNull: true })
+  acceptedAt: Date
```
Comment (tan-zx) [inline backend/src/sms/models/sms-message-transactional.ts:60]: if `allowNull` is `true`, the type here should be `Date | null` too

Comment (richard-yak) [reply backend/src/sms/models/sms-message-transactional.ts:60]: missed that out, added

**Review by stanleynguyen — APPROVED** [review]

**Review by stanleynguyen — CHANGES_REQUESTED** [review]
eh wait sorry, wrong review

**Review by stanleynguyen — APPROVED** [review]

**Pull request merged**